src/main/kotlin/CheckString.kt

```diff
@@ -0,0 +1,20 @@
+object CheckString {
+    fun checkBlank(input: String) {
+        if (input.equals(" ") || input.isNullOrEmpty()) {
+            throw IllegalArgumentException("공백은 입력할 수 없습니다.")
+        }
+    }
+
+    fun checkOperator(splitString: List<String>): Boolean {
+        val operator = listOf("+", "-", "*", "/")
+        val eachSplitString = splitString.indices
+        for (i in eachSplitString) {
```

Comment:
```suggestion
        for (i in splitString.indices) {
```
이건 인라인으로 써도 가독성을 해치지 않을거같아.